web/sites/default/config/editor.editor.simplified_editor_with_media_embed.yml

```diff
@@ -0,0 +1,65 @@
+uuid: 9ae21b4e-957c-4762-b729-d30661bb0568
+langcode: en
+status: true
+dependencies:
+  config:
+    - filter.format.simplified_editor_with_media_embed
+  module:
+    - ckeditor
+format: simplified_editor_with_media_embed
+editor: ckeditor
+settings:
+  toolbar:
+    rows:
+      -
+        -
+          name: Formatting
+          items:
+            - Bold
+            - Italic
+            - Blockquote
+            - Format
+            - Styles
+        -
+          name: Lists
+          items:
+            - BulletedList
+            - NumberedList
+        -
+          name: Links
```

Comment:
Is this still going to use the LinkIT profile? We need to make sure and test for that in QA.

Reply:
Ahhhh, I missed the LinkIT config for the new editor. I'll add that.